addons/hr_homeworking/static/src/components/hr_presence_status/hr_presence_status.js

```diff
@@ -9,7 +9,7 @@ const patchHrPresenceStatus = () => ({
     get color() {
         if (this.location) {
             let color = "text-muted";
-            if (this.props.record.data.hr_presence_state !== "to_define") {
+            if (this.props.record.data.hr_presence_state !== "to_define" && this.props.record.data.hr_presence_state !== "offline") {
```

Comment:
According to the odoo js linter:
```suggestion
            if (
                this.props.record.data.hr_presence_state !== "to_define" &&
                this.props.record.data.hr_presence_state !== "offline"
            ) {
```